nuimo/src/androidTest/kotlin/com/senic/nuimo/NuimoBluetoothControllerTest.kt

```diff
@@ -13,6 +13,8 @@ import kotlin.concurrent.schedule
 //TODO: Try spek test framework: http://jetbrains.github.io/spek/
 //TODO: Add timeouts to each test
 class NuimoBluetoothControllerTest: NuimoDiscoveryManagerTest() {
+    val DIRECTION_CLOCKWISE = 1;
+    val DIRECTION_COUNTER_CLOCKWISE = -1;
```

Comment:
- Please remove `;` and move both constants as variables (no uppercase) into the test case's method. These constants are only needed in a single test and thus shouldn't be exposed to the class. Thus names should be `directionClockwise` and `directionCounterclockwise`. (Counterclockwise is a single word ;)
- Please rename argument `swipeDirection` to `rotationDirection` and merge your changes into the existing commit.